pytorch_lightning/accelerators/ddp_cpu_slurm_accelerator.py

```diff
@@ -15,16 +15,15 @@
 from typing import List, Optional
 
 import torch
-import torch.distributed as torch_distrib
 import torch.distributed as dist
```

Comment:
@SeanNaren @williamFalcon `import torch.distributed as dist` was picked due to alphabetical nature. if the rest of the codebase uses `torch_distrib` instead, I can change this

Reply:
we can do a follow up PR to make sure when using torch dist we abbreviate it as that so we don't collide with our dist